etc/kayobe/cephadm.yml

```diff
@@ -91,7 +91,7 @@ kolla_ceph_services:
 kolla_ceph_cinder_volume_required: "{{ kolla_enable_cinder | bool }}"
 
 # Whether to generate Ceph configuration for Cinder backup.
-kolla_ceph_cinder_backup_required: "{{ kolla_enable_cinder_backup | default(false) | bool }}"
+kolla_ceph_cinder_backup_required: "{{ kolla_enable_cinder_backup | default(true) | bool }}"
```

Comment:
This change was already made into a [separate PR](https://github.com/stackhpc/stackhpc-kayobe-config/pull/243) and has since been merged into stackhpc/xena and stackhpc/yoga by the look of it